src/browser/modules/Main/Main.tsx

```diff
@@ -83,8 +81,6 @@ const Main = React.memo(function Main(props: any) {
           Server is taking a long time to respond...
         </WarningBanner>
       )}
-      {props.utilizeBrowserSync && <SyncReminderBanner />}
```

Comment:
Seems like this prop is still being passed to the component even though not used anymore?